eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/MultiReaderFastList.java

```diff
@@ -1419,6 +1454,15 @@ public void forEachWithIndex(int fromIndex, int toIndex, ObjectIntProcedure<? su
             return this.getDelegate().selectInstancesOf(clazz);
         }
 
+        /**
+         * @since 10.0
+         */
+        @Override
+        public void sort(Comparator<? super T> comparator)
+        {
+            this.getDelegate().sort(comparator);
+        }
+
         @Override
         public MutableList<T> sortThis()
```

Comment:
I see sort and sortThis together in the inner class. They should be together in the outer class too.